stacker/providers/aws/default.py

```diff
@@ -15,6 +15,13 @@
 MAX_TAIL_RETRIES = 5
 
 
+def template_args(template):
```

Comment:
> Also - any reason to not make this a method of the Template object itself?

Mainly because it's specific to the AWS implementation of the provider, whereas `stacker.providers.base` isn't implementation specific.